models/issues/issue_list.go

```diff
@@ -230,11 +230,11 @@ func (issues IssueList) loadMilestones(ctx context.Context) error {
 }
 
 func (issues IssueList) getProjectIDs() []int64 {
-	ids := make(container.Set[int64], len(issues))
+	var ids []int64
 	for _, issue := range issues {
-		ids.Add(issue.ProjectID())
+		ids = append(ids, issue.ProjectIDs()...)
 	}
-	return ids.Values()
+	return ids
```

Comment:
Why it should use duplicate IDs in the list?

Reply:
Modified